api/proto/info/info.proto

```diff
@@ -618,12 +703,26 @@ message ZInfoVolume {
 
   ErrorInfo volumeErr = 7;
   int64 generation_count = 8; // version of volume
+  ZVolumeRunState run_state = 9;
```

Comment:
We already report ZSwState for volumes, so no need to introduce a new enum here.
We should document the set of ZSwStates which are valid in VolumeStatus but that does not require an API transition/deprecation.

Reply:
Is it not better to get rid of ZSwState Once and for all? We are doing the work for all other objects anyways. Why retain that monolithic Enum?

Reply:
What problem are you solving by merely renaming things? ZSwState has all the states needed for application instances and it makes zero sense to deprecate that.

Since volumes and content trees share >50% os those states *and* we already report ZSwState for them, it makes no sense to me to force an API transition to a different field containing the same information (and with different names).

The problem we needed to solve was to introduce state reporting where there was none (for devices and network instances). Renaming existing state is not desirable due to the API transition pain it will inflict.